templates/_navigation.html.twig

```diff
@@ -0,0 +1,9 @@
+{% block navigation %}
+    <footer>
+        <div class="navbar">
```

Comment:
footer is like a div, but with a semantic. so your code is like:

    <div>
        <div>
            something
        </div>
    <div>

try:

    <footer class="flex-row justify-center">
        <a href="...">...</a>
    </footer>